ecm_prep.py

```diff
@@ -13742,94 +13360,10 @@ def main(base_dir):
-    parser.add_argument("--pkg_env_costs", action="store_true",
-                        help=("Account for envelope costs in HVAC/envelope "
-                              "packages (and their subsequent competition)"))
-    # Optional flag to handle fuel switching conversions via exogenous rates
-    parser.add_argument("--exog_hp_rates", action="store_true",
-                        help=("Accomodates exogenous forcing of fuel "
-                              "switching conversion rates"))
-    # Optional flag that will set baseline electricity emissions intensities to
-    # be consistent with the GridSIM Reference Case (rather than AEO)
-    parser.add_argument("--gs_ref_carb", action="store_true",
-                        help=("Use GridSIM Reference Case electricity "
-                              "emissions intensities"))
-    # Optional flag to use alternate grid decarbonization case
-    parser.add_argument("--grid_decarb", action="store_true",
-                        help="Flag alternate grid decarbonization scenario")
-    # Optional flag to restrict adoption schemes
-    parser.add_argument("--adopt_scn_restrict", action="store_true",
-                        help="Restrict to a single adoption scenario")
-    # Optional flag to force early retrofit rate to zero
-    parser.add_argument("--retro_set", action="store_true",
-                        help="Prompt user for early retrofit rate settings")
-    # Optional flag to add typical efficiency tech. analogues for ESTAR, IECC,
-    # or 90.1 measures (to account for competitive effects)
-    parser.add_argument("--add_typ_eff", action="store_true",
-                        help=("Add typical technology analogues for equipment "
-                              "efficiency measures at the most current ENERGY "
-                              "STAR, IECC, and 90.1 performance levels"))
-    # Optional flag that will develop copies HVAC/envelope packages that remove
-    # the relative impacts of the HVAC portion to isolate the impacts of the
-    # envelope portion
-    parser.add_argument("--pkg_env_sep", action="store_true",
-                        help=("Isolate the impacts of envelope in "
-                              "envelope + HVAC packages"))
-    # Optional flag to use more detailed building type breakouts
-    parser.add_argument("--detail_brkout", action="store_true",
-                        help=("Use more detailed region/building type "
-                              "breakouts as applicable"))
-    # Optional flag to include accounting of fugitive emissions from
-    # refrigerant leakage and/or supply chain methane leakage
-    parser.add_argument("--fugitive_emissions", action="store_true",
-                        help="Account for fugitive emissions sources")
-    # Object to store all user-specified execution arguments
-    opts = parser.parse_args()
+    opts = ecm_args(sys.argv[1:])
 
     # Set current working directory
```

Comment:
Please delete this comment line or move it to before line 12574, where the getcwd() call is now located?